testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -223,31 +207,21 @@ def test_ensemble_superimposition():
 
     @staticmethod
     def test_ensemble_superimposition_to_reference_non_weighted():
-        ensemble0 = mda.Universe(PSF, DCD)
```

Comment:
ensemble0 was actually not being used later on in the assert_equal - it was probably some leftover for debugging purposes, so it didn't make much sense to keep it